config/config-logging.yaml

```diff
@@ -10,24 +10,7 @@ metadata:
     bindings.labs.vmware.com/release: devel
 
 data:
-  _example: |
```

Comment:
it's worth keeping the example as a representation of what the default values are, and what fields are available.

Reply:
example added